components/brave_shields/browser/filter_list_catalog_entry.cc

```diff
@@ -54,15 +84,19 @@ FilterListCatalogEntry::FilterListCatalogEntry(
     const std::string& support_url,
     const std::string& component_id,
     const std::string& base64_public_key,
-    const std::string& desc)
+    const std::string& desc,
+    const std::string& ios_component_id,
```

Comment:
I don't understand why we have a separate entry for ios_component_id. Shouldn't we just initialize this with the correct component id for ios vs desktop?

Reply:
See https://github.com/brave/brave-core/pull/15077#discussion_r976700939 - this struct is just a 1:1 C++ representation of what we're shipping from the component server and we'll remove the `ios_` fields altogether once https://github.com/brave/brave-ios/issues/5975 is finished